### PR TITLE
Add details for Cloud forking

### LINKED ISSUE #1048: Clarify details on forking
# Describe change in content, appearance, or functionality

Clarify forking details, especially whether setup for the fork taxes the running instance.

[Relevant Slack thread](https://timescaledb.slack.com/archives/C02GZAT5LKD/p1651175486802089)

# Subject matter expert (SME)

[If known, who is a good person to ask about this topic]

# Deadline

[When does this need to be addressed]

# Any further info

[Anything else you want to add, or further links]

### PR DESCRIPTION
**Pull request by charislam**
# Description

Clarify that forking doesn't tax the running instance, in case readers are worried about overloading the source database

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #1048

### CODE REVIEW

File: cloud/service-operations/general.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ testing purposes, or to prepare for a major version upgrade. The only difference
 between the original and the forked service is that the `tsdbadmin` user has a
 different password.
 
+The fork is created by restoring from backup and applying the Write-Ahead Log.
```
Comment (Loquacity) [inline cloud/service-operations/general.md:12]: Do we capitalise write-ahead log elsewhere?

Comment (charislam) [reply cloud/service-operations/general.md:12]: Good question. We seem to be inconsistent. Do you have a preference? I can go through the docs and update everything to one style (capitalized or not)

Comment (Loquacity) [reply cloud/service-operations/general.md:12]: I prefer fewer initcaps, generally. In the backups section, I used lowercase in the index (https://docs.timescale.com/timescaledb/latest/how-to-guides/backup-and-restore/) but then basically referred to WAL-E in the main section. 

image

I think we can probably ditch the initcaps.

Comment (charislam) [inline cloud/service-operations/general.md:12]:
```suggestion
The fork is created by restoring from backup and applying the write-ahead log.
```

**Review by Loquacity — APPROVED** [review]

**Pull request merged**